### PR TITLE
qemu.tests: use snapshot to avoid backup and retore steps

### PR DESCRIPTION
**Pull request by xutian**
enable snapshot for disk image, and commit changes when OS updated.
so that we can skip logical about backup and restore OS.

ID: 1394120

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by suqinhuang** [comment]
bug id?

File: qemu/tests/rh_kernel_update.py [hunk 1]
```diff
@@ -20,19 +22,15 @@ def run(test, params, env):
...
-    17) Verifying the virtio drivers (Optional)
+    5) Install guest kernel firmware (Optional)
+    6) Install guest kernel
+    7) Install guest kernel debuginfo (Optional)
+    8) Installing virtio driver (Optional)
+    9) Update initrd file
+    10) Make the new installed kernel as default
+    11) Update the guest kernel cmdline (Optional)
+    12) Reboot guest after updating kernel
+    13) Verifying the virtio drivers (Optional)
```
Comment (suqinhuang) [inline qemu/tests/rh_kernel_update.py]: There should be a last step, commit the update

**Comment by suqinhuang** [comment]
Others look ok to me
PASS 1-smp_1.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.rh_kernel_update

**Comment by xutian** [comment]
Merged, thanks @suqinhuang

**Comment by suqinhuang** [comment]
Hi xutian, you lost one step description
14). commit the update

**Comment by CongLi** [comment]
There is typo in the commit log.
s/retore/restore

**Pull request merged**